docs/UserGuide.md

```diff
@@ -376,6 +376,7 @@ These are a list of features that help you manage the records of appointment:
 - [Listing appointment](#listing-all-appointments-or-by-patient-or-doctor-nric--lista)
 - [Viewing appointment](#viewing-an-appointment--viewa)
 - [Editing appointment](#editing-a-appointment--edita)
+- [Adding a remark to an appointment](#deleting-an-appointment--deletea)
```

Comment:
- [Adding a remark to an appointment]**(#deleting-an-appointment--deletea)**
need to change the behind part sirr.